src/platform/intel/cavs/dai.c

```diff
@@ -232,8 +127,20 @@ int dai_init(void)
 	int i;
 
 	/* init ssp */
-	/* TODO: move all the properties initialization here */
 	for (i = 0; i < ARRAY_SIZE(ssp); i++) {
+		ssp[i].type = SOF_DAI_INTEL_SSP;
```

Comment:
@lgirdwood The descriptors (above) are still created during the platform initialization to be registered in the lib layer. Since they are statically defined atm, they are just moved from .data to .bss by this patch. The platform headers define the quantity. This is a step towards dynamic allocation of the descriptors from the heap during run-time discovery (using HW caps registers).

Reply:
ok, btw - looks like there is a conflict now